packages/reward-root-submitter/reward_root_submitter/main.py

```diff
@@ -25,7 +25,7 @@ class MerkleRoot:
 
 
 def setup_logging(config):
-    logging.basicConfig(level=config.log_level.upper())
+    logging.getLogger().setLevel(level=config.log_level.upper())
```

Comment:
https://stackoverflow.com/questions/32681289/python-logging-wont-set-a-logging-level-using-basicconfig

Need to put this `basicConfig` at top of file, or use `setLevel`. Chose latter